cli/azd/cmd/deploy.go

```diff
@@ -154,7 +154,7 @@ func (d *deployAction) Run(ctx context.Context, cmd *cobra.Command, args []strin
 			fmt.Println(deployMsg)
 			spinner := spin.NewSpinner(deployMsg)
 			spinner.Start()
-			err := deployAndReportProgress(spinner.Title)
+			err = deployAndReportProgress(spinner.Title)
```

Comment:
Looks good - can you see if this pattern is used in any of the other commands where we are using the Spinner to ensure we didn't break it somewhere else also?

Reply:
Other spinner use cases look good, in particular the one in `infra create`, which uses similar code.